.github/workflows/workflow-branch.yml

```diff
@@ -0,0 +1,39 @@
+# Workflow that runs only on internal branches that are not then main one.
+# Triggered by pushes on internal branches (excluding main and tags).
+# Build, test, run coverage analysis and release to Maven Central a final release.
+name: Workflow on internal branch, excluding main
+
+on:
+  push:
+    branches-ignore:
+      - 'main'
+      - 'TrueLayer:**' #TODO: remove
```

Comment:
Is this meant to be here? Given the TODO

Reply:
Leftover: thanks! sorted